product_docs/docs/pem/8/installing/configuring_the_pem_server_on_linux.mdx

```diff
@@ -52,6 +52,30 @@ After selecting a configuration option, the script prompts you for configuration
 /usr/edb/pem/bin/configure-pem-server.sh -help
 ```
 
+## Post-configuration steps when web server and PEM backend database are installed separately
+
+If you choose to run the web application server on a separate host from the backend database, you need to perform some additional manual steps before PEM is fully operational.
+
+Make sure that the backend Postgres database accepts the connections from any user permitted to log in to PEM from the web application server. To achieve this, add this entry to `pg_hba.conf`: 
+
+```ini
+host pem +pem_user <web_app_ip>/32 md5
+host postgres +pem_user <web_app_ip>/32 md5
```

Comment:
In general - we don't need permission to the `postgres` database.
They need permission to 'postgres' database only if they would like to acccess it from the PEM GUI (PEM backend database server access).